prime-router/docs/troubleshooting.md

```diff
@@ -0,0 +1,22 @@
+# Troubleshooting/Production Support
+
+Note this documentation does not currently exist but should be created.
+
+## Common Failures & Fixes
```

Comment:
I am not sure that this should be in senders. I think troubleshooting warrants its own directory. Also we should probably take out the cuss words from this.

Reply:
Will remove the language, not necessarily following the senders bit?

Reply:
Sorry about the senders bit. I think I was confusing something. Language still needs to be removed though.

Reply:
Yep dropped the language